app/src/main/java/cm/aptoide/pt/download/DownloadAnalytics.java

```diff
@@ -164,17 +166,22 @@ private void sendDownloadEvent(String downloadCacheKey) {
   }
 
   public void downloadStartEvent(Download download, AnalyticsManager.Action action,
-      AppContext context) {
-    downloadStartEvent(download, 0, null, context, action);
+      AppContext context, Boolean isMigration) {
+    downloadStartEvent(download, 0, null, context, action, isMigration);
   }
 
   public void downloadStartEvent(Download download, int campaignId, String abTestGroup,
-      AppContext context, AnalyticsManager.Action action) {
+      AppContext context, AnalyticsManager.Action action, boolean isMigration) {
     Map<String, Object> event = new HashMap<>();
     ScreenTagHistory screenTagHistory = navigationTracker.getPreviousScreen();
     event.put(APP, createAppData(download));
     event.put(NETWORK, AptoideUtils.SystemU.getConnectionType(connectivityManager)
         .toUpperCase());
+    if (isMigration) {
+      event.put(ORIGIN, UPDATE_TO_APPC);
+    } else {
+      event.put(ORIGIN, getOrigin(download));
+    }
```

Comment:
Putting 2 times the same parameter in case it is not a migration:
- in the else clause and the line after

Reply:
fixed